BitmovinAnalyticsCollector/Classes/Collector/data/SimpleEventDataDispatcher.swift

```diff
@@ -20,9 +20,9 @@ class SimpleEventDataDispatcher: EventDataDispatcher {
                 guard let events = self?.events.enumerated().reversed() else {
                     return
                 }
-                for (i, eventData) in events {
+                for (imdex, eventData) in events {
```

Comment:
shouldn't this be `index`?

Reply:
yes, it's a typo, should be index. Swiftlint demands variable names not shorter than 3 chars. So, you can name it anything longer than in ;)